src/main/java/org/ibp/api/rest/germplasm/GermplasmPedigreeResource.java

```diff
@@ -0,0 +1,86 @@
+package org.ibp.api.rest.germplasm;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import org.generationcp.middleware.api.germplasm.pedigree.GermplasmNeighborhoodNode;
+import org.generationcp.middleware.api.germplasm.pedigree.GermplasmTreeNode;
+import org.generationcp.middleware.domain.germplasm.GermplasmDto;
+import org.ibp.api.java.germplasm.GermplasmPedigreeService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import javax.annotation.Resource;
+import java.util.List;
+
+@Api(value = "Germplasm Pedigree Services")
+@Controller
+public class GermplasmPedigreeResource {
+
+	@Resource
+	private GermplasmPedigreeService germplasmPedigreeService;
+
+	@ApiOperation(value = "Returns the of the given germplasm")
```

Comment:
Returns the **pedigree tree** of the given germplasm

Reply:
Done